includes/class-wc-skroutz-analytics-tracking.php

```diff
@@ -248,4 +253,22 @@ private function calculate_tax_from_total($total, $tax_rate) {
 		return round( $total - $total / ( 1 + $tax_rate/100 ), 2 );
 	}
 
+	/**
+	 * Checks if order is prior to 30 days ago
+	 *
+	 * @return boolean
+	 *
+	 * @since    1.6.9
+	 * @access   private
+	 */
+	private function order_is_obsolete() {
+		$thirty_days_ago = (new DateTime())->modify("-30 day");
+
+		if ( is_null($this->order->get_date_created()) ) {
+			return false;
+		}
+
+		$order_creation_date = new DateTime($this->order->get_date_created()->date('Y-m-d H:i:s'));
+		return ($thirty_days_ago > $order_creation_date);
```

Comment:
Also `get_date_created` returns a WC_DateTime which inherits DateTime. I think we can just do:

```php
return (new DateTime('- 30 days', wp_timezone())) > $this->order->get_date_created();
```

But we will have to check backwards compatibility for this.